api/swagger-spec/v1.json

```diff
@@ -5971,7 +5971,7 @@
         "allowMultiple": false
        },
        {
-        "type": "",
+        "type": "string",
```

Comment:
@bgrant0607 for swagger spec 1.2, you can use `allowMultiple` for query parameter to represent multiple values:

https://github.com/swagger-api/swagger-spec/blob/master/versions/1.2.md#524-parameter-object